io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -958,9 +979,15 @@ impl NvmfSubsystem {
     }
 
     /// get ANA state
+    /// XXX: The SPDK NVME multipath is transport protocol independent. Should
+    /// we keep returning the ana state here by default using TCP transport
+    /// as today?
```

Comment:
does this mean, that we should instead return multiple states, one per transport type?